bank_management.cpp

```diff
@@ -0,0 +1,115 @@
+// C++ program to implement

```

Comment:
You need to have this file put under the `Task 3` folder. That's about it!

Reply:
@nimishbongale  thanks, I have reuploaded with correct structure . Please consider.